recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -56,7 +56,7 @@ CMAKE_QT5_OECONF = "\
 
 export WK_USE_CCACHE="NO"
 
-PACKAGECONFIG ??= "jit dfg-jit mediasource video webaudio webcrypto woff2 gst_gl \
+PACKAGECONFIG ??= "jit dfg-jit mediasource video webcrypto woff2 gst_gl \
```

Comment:
what is the reason for removing webaudio? did it fail to build?

Reply:
There was an issue in the gstreamer webaudio dependencie. Some broken api if I remember well. But I removed it as I don't have an audio device anyway.